src/containers/homepage/survey-list.component.tsx

```diff
@@ -1,17 +1,40 @@
-import { Button, Table } from 'antd';
+import { Button, Col, Icon, Row, Table, Tooltip } from 'antd';
+import Search from 'antd/lib/input/Search';
+import { RadioChangeEvent } from 'antd/lib/radio';
+import RadioGroup from 'antd/lib/radio/group';
+import RadioButton from 'antd/lib/radio/radioButton';
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 import { StatusTag } from '../../common/components/status-tag/status-tag.component';
+import { mapStatusToIcon } from '../../common/mappers/survey-status.mapper';
 import { SurveyStatus } from '../../common/types/survey-status.type';
-import { Survey } from '../../common/types/survey.type';
+import { Survey, surveyDateToLocaleString } from '../../common/types/survey.type';
 
-export interface SurveyListComponentProps {
+export interface SurveyListProps {
     items: Survey[];
 }
 
-// TODO: display some data
-export class SurveyListComponent extends Component<SurveyListComponentProps> {
+export class SurveyList extends Component<SurveyListProps> {
```

Comment:
Noticing some inconsistencies here - the file `survey-list.component.tsx` lives in a `containers` folder and the exported class is `SurveyList` (without a suffix)

Reply:
@decrn asked for this. Status tag isn't suffixed with Component either. The reasoning behind it was he doesn't like long classnames. At the moment Containers are suffixed but Components aren't.

Reply:
Where do you suggest placing the SurveyList component if not in the directory of the container where it is used?

I also don't like the use of "SurveyListComponent", because it being a component is implied. After all, it inherits from `React.Component` .

Containers I would suffix because it is not inherently implied that they are containers. Also, are containers ever used inside of other containers?